.github/workflows/CI-build-test.yml

```diff
@@ -47,28 +47,20 @@ jobs:
           echo "USER_ID=$(id -u)" >> $GITHUB_ENV
           echo "GRP_ID=$(id -g)" >> $GITHUB_ENV
 
-      - name: Cargo check
-        run: ${{ env.DOCKER_COMPOSE_CMD }} cargo check --all-features --all-targets
-
       - name: Cargo build
         run: ${{ env.DOCKER_COMPOSE_CMD }} cargo build --release
 
       - name: Cargo unit tests
         shell: bash
         run: ${{ env.DOCKER_COMPOSE_CMD }} cargo test --lib --bins --all-features --no-fail-fast --release 2>&1 | tee unit_tests_output.txt
 
-      - name: Cargo integration tests
-        shell: bash
-        run: ${{ env.DOCKER_COMPOSE_CMD }} cargo test --test '*' --all-features --no-fail-fast --release 2>&1 | tee integration_tests_output.txt
-
```

Comment:
In the relay chain implementation that I'm doing I've integrated some integration tests from polkadot... ok maybe we can live without them but I prefer to recover it.

Reply:
Ok reverted, integration tests are now back again.
Force pushing.